### PR TITLE
[FABCAG-5] Add tutorials and checks

### PR DESCRIPTION
**Pull request by awjh-ibm**
- Add three tutorials
- Update readme to include link to tutorials
- Add script to check all tutorials content sections have valid sections

### CODE REVIEW

File: .azure-pipelines/scripts/tutorial-checks.sh [hunk 1]
```diff
@@ -0,0 +1,98 @@
+#!/bin/bash
```
Comment (heatherlp) [inline .azure-pipelines/scripts/tutorial-checks.sh]: Can you add some comments into this script explaining what the functions are doing please? It's not immediately clear what it's checking for

File: README.md [hunk 2]
```diff
@@ -23,6 +23,11 @@ There’s always plenty to do! Check the documentation on
 ## Documentation
 Documentation for all packages within this repo can be found [here](http://godoc.org/github.com/hyperledger/fabric-contract-api-go)
 
+## Tutorial
+Tutorials for how to be smart contracts using packages in this repo can be [here](./tutorials)
```
Comment (heatherlp) [inline README.md]: do you want to be a smart contract too? :D

File: tutorials/README.md [hunk 3]
```diff
@@ -0,0 +1,8 @@
+# Tutorials
+
+This repo contains three tutorials to guide you the very basics up to advanced use of packages in this repo.
```
Comment (heatherlp) [inline tutorials/README.md]: "guide you the very basics"

File: tutorials/getting-started.md [hunk 4]
```diff
@@ -0,0 +1,359 @@
+# Getting started
+
+## Tutorial contents
+- [Prerequisites](#prerequisites)
+- [Housekeeping](#housekeeping)
+- [Declaring a contract](#declaring-a-contract)
+- [Writing contract functions](#writing-contract-functions)
+- [Using contracts in chaincode](#using-contracts-in-chaincode)
+- [Testing your chaincode as a developer](#testing-your-chaincode-as-a-developer)
+- [What to do next?](#what-to-do-next?)
```
Comment (heatherlp) [inline tutorials/getting-started.md]: if you preview this file, the what to do next link doesn't take you anywhere

**Comment by awjh-ibm** [comment]
Updated changes include spelling/grammar fixes for the above, removal of punctuation from contents pages and comments in tutorial checking script

File: tutorials/using-advanced-features.md [hunk 5]
```diff
@@ -0,0 +1,200 @@
...
+```
+
+with:
+```
+existing := ctx.GetData()
+```
+
+> Note: As you have removed the definition of err you will need to change err = to err := in the Create and Update functions
+
+Your chaincode should now work exactly the same as before. If you have torn down your network from the previous tutorial you can run through the steps outlined [there](./getting-started.md#testing-your-chaincode-as-a-developer) to bring your chaincode up and interact with it. Otherwise stop the chaincode process, rebuild your go program and restart it using the same command. You can then interact with the same invoke/query commands.
```
[the remark this answers is not in the document]
Comment (heatherlp) [reply tutorials/using-advanced-features.md]: there's another one in the third tutorial too

**Comment by awjh-ibm** [comment]
All comments at this point resolved by last force push

**Pull request merged**